front/src/components/AdminPanel/Panel/Categories/Categories.js

```diff
@@ -1,8 +1,292 @@
+        id: 4,
+        name: "juniorzy",
+    },
+];
+
+const distances = [
+    {
+        distance: 90,
+        seriesAmount: 12,
+        seriesType: "3-strzałowe",
+    },
+    {
+        distance: 70,
+        seriesAmount: 6,
+        seriesType: "6-strzałowe",
+    },
+];
+
+function clearRadioButtons(radioName) {
+    var buttons = document.getElementsByName(radioName);
+    for (var i = 0; i < buttons.length; i++) {
+        buttons[i].checked = false;
+    }
+}
+
+const Categories = () => {
+    const [categoryName, setCategoryName] = useState("");
+    const [categoryGender, setCategoryGender] = useState("");
+    const [distance, setDistance] = useState("");
+    const [seriesAmount, setSeriesAmount] = useState("");
+    const [seriesType, setSeriesType] = useState("");
+    const [chosenCategory = false, setChosenCategory] = useState("");
+    const [chosenDistance = false, setChosenDistance] = useState("");
+
     return (
         <div>
-            <p>Kategorie</p>
+            <p className="panel-detail-header">Kategorie </p>
+            <div className="infoDiv">
+                <div className="category-column-div">
+                    <div className="category-tile-div">
+                        <p className="categories-header"> Nowa kategoria </p>
+                        <form>
+                            <input
+                                type="text"
+                                placeholder="Nazwa kategorii"
+                                className="form-control"
+                                value={categoryName}
+                                onChange={(event) =>
+                                    setCategoryName(event.target.value)
+                                }
+                            />{" "}
+                            <br />
+                            <select
+                                name="gender"
+                                id="gender"
+                                value={categoryGender}
+                                className="form-control"
+                                onChange={(event) =>
+                                    setCategoryGender(event.target.value)
+                                }
+                            >
+                                <option value="women" key="women">
+                                    {" "}
+                                    Kobiety{" "}
+                                </option>
+                                <option value="men" key="men">
+                                    {" "}
+                                    Mężczyźni{" "}
+                                </option>
+                            </select>{" "}
+                        </form>
+                        <br />
+                        <Button
+                            type="submit"
+                            placeholder="Dodaj kategorię"
+                            className="btn btn-primary btn-lg ml-5"
+                        />
```

Comment:
I'm not sure if that was intended but moving Button out of form results in onSubmit event not being called. Applies to Button in line 201 as well